.github/workflows/testing.yml

```diff
@@ -8,22 +8,26 @@ jobs:
       # Checks out a copy of your repository
       - name: Checkout repository
         uses: actions/checkout@v1
-      - name: Use Xcode 12.2
-        run: sudo xcode-select -switch /Applications/Xcode_12.2.app
-      - name: Install CocoaPod dependencies
-        run: pod install
+      - name: Use Xcode 12.3
+        run: sudo xcode-select -switch /Applications/Xcode_12.3.app
+      - name: Install xcpretty
+        run: gem install xcpretty
+      - name: Install Carthage dependencies
+        run: cd Demo && ../carthage.sh bootstrap
       - name: Run Unit Tests
-        run: set -o pipefail && xcodebuild -workspace 'BraintreeDropIn.xcworkspace' -sdk 'iphonesimulator' -configuration 'Release' -scheme 'UnitTests' -destination 'name=iPhone 11,platform=iOS Simulator'  test  | ./Pods/xcbeautify/xcbeautify
+        run: set -o pipefail && xcodebuild -workspace 'BraintreeDropIn.xcworkspace' -sdk 'iphonesimulator' -configuration 'Debug' -scheme 'UnitTests' -destination 'name=iPhone 11,platform=iOS Simulator'  test  | xcpretty
```

Comment:
curious: is `Debug` more favorable for unit testing for the Drop-In project?

Reply:
Good question. So running in `Debug` mode will build solely for the simulator architecture, whereas `Release` will build for all potential architectures (device & simulator).

But when running in `Release` mode we get the error: `building for iOS Simulator, but linking for architecture arm64`.

Since Xcode 12 introduces arm64 as a valid device architecture for Apple Silicon Macs, and it is also a valid simulator architecture - I think Xcode is getting confused on which one it is building for. Also, our Intel macs don't offer arm64 simulator support.

Reply:
It would have been nice if Apple named their new Apple Silicon architecture to arm64S or something. With XCode 12 we're having to do a handful of weird workarounds for this new "feature". So for now - running our tests in Debug works.

We confirmed everything runs fine on an actual device, too.